unity/Assets/Scripts/Logic/Hand.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Hexxle.Interfaces;
+using Hexxle.Unity;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Hexxle.Logic
+{
+    public class Hand
+    {
+        private List<ITile> Slots;
```

Comment:
Any reason for capitalized variables? I thought the convention said any non-contant variables use camelCase

Reply:
I completely mixed that up..., should be fixed now